src/main/java/com/truelayer/java/TrueLayerClientBuilder.java

```diff
@@ -182,12 +185,13 @@ public TrueLayerClientBuilder withHttpLogs(Consumer<String> logConsumer) {
      * @return the instance of the client builder used
      */
     public TrueLayerClientBuilder withCredentialsCaching() {
-        this.credentialsCache = new SimpleCredentialsCache(Clock.systemUTC());
+        this.credentialsCache = new InMemoryCredentialsCache(Clock.systemUTC());
```

Comment:
As we anyway need a breaking change for the new cache interface (that aligns to what PHP and .NET libs do), I took the chance to rename this to be more explicit